src/go/github-runner-monitor/cmd/root.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright © 2023 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cmd handles the command lines.
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/github-runner-monitor/pkg/monitor"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/sys/unix"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "github-runner-linux",
+	Short: "Manage ephemeral GitHub runners for Rancher Desktop",
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		logrus.SetLevel(logrus.InfoLevel + logrus.Level(viper.GetInt("verbose")))
+		logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
+	},
+	Args: cobra.ExactArgs(0),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+		ctx, _ := signal.NotifyContext(cmd.Context(), unix.SIGINT, unix.SIGTERM, unix.SIGHUP)
```

Comment:
Does this need `defer stop()` here?

Reply:
Not really (the process exits right after), but added anyway.